src/components/RadioButtonList/RadioButtonList.tsx

```diff
@@ -139,7 +197,9 @@ const RadioButtonList = <T extends RadioButtonOption>({
         id={id}
         {...rest}
       >
-        {radioOptions.map((option, index) => renderRadioButton(option, index))}
+        {radioOptions.map((option, index) =>
+          renderRadioButton(option, index, formValidation)
```

Comment:
I see this was already a function, but would it not be better just add jsx here? or make a special Button component within RAdioButtonList/

Reply:
You're right!